### ISSUE TITLE
After upgrade of Umbraco 7.5.3 -> 7.5.4 LeBlender can't find datatype

### ISSUE THREAD

**Issue by bjarnef** [Lecoati/LeBlender#53]
After Umbraco Cloud has upgraded from 7.5.3 to 7.5.4, it seems it broke LeBlender.

E.g. an error in console like:
```
Failed to load resource: the server responded with a status of 417 (Missing token)

/umbraco/backoffice/LeBlenderApi/DataType/GetPropertyEditors?guid=0cc0eba1-9960-42c9-bf9b-60e150b429ae
```` 

From the commit in UaaS / Umbraco Cloud AutoMapper has beeen modified, which could be the reason why it fails somewhere in this method:
https://github.com/Lecoati/LeBlender/blob/09fb8f87bbe9a714b40cc37a0fae816a24b019f4/Src/Lecoati.LeBlender.Extension/Controllers/DataTypeController.cs#L40-L53

**Comment by bjarnef** [comment]
After spending some time on this, I compared with another site on UaaS / Umbraco Cloud, which also run on v7.5.4.

Then I noticed this difference in web.config, between this site (left side) and the other site (right side) where it works.

![image](https://cloud.githubusercontent.com/assets/2919859/19978030/d53b264e-a1f5-11e6-9acc-be0e422e2bdc.png)

I just removed xml namespace from AutoMapper - after this change and commit to UaaS, LeBlender now works.

Funny enough UaaS did upgrade both sites automatically from 7.5.3 -> 7.5.4 - don't know why there is this difference.

https://twitter.com/bfyrstenborg/status/794232678509723648
